bot/constants.py

```diff
@@ -16,7 +17,7 @@ class AdventOfCode:
     leaderboard_id = 363275
     leaderboard_join_code = str(environ.get("AOC_JOIN_CODE", None))
     leaderboard_max_displayed_members = 10
-    year = 2018
+    year = datetime.utcnow().year
```

Comment:
For local debugging it might be nice to have an env var to override this if it's being worked on after the New Year but before December 1st.

Reply:
but isnt the command linked to the season that ends at new year?

Reply:
Yes, but we have commands to manually load seasons so you can easily activate one outside of its regular time period. An optional environment variable with the current year as the default value would work well for that since you'd be able to work with last year's edition in lieu of the not-yet-available edition of the current year.

Reply:
I needed this change for debugging so I went ahead and made it